styles/Datadog/time.yml

```
@@ -0,0 +1,9 @@
+extends: existence
+message: "Format times as 'HOUR:MINUTE a.m.' or HOUR:MINUTE p.m.' instead of '%s'."
+link: "https://datadoghq.atlassian.net/wiki/spaces/WRIT/pages/2732523547/Style+guide#%s"
+level: error
+nonword: true
+tokens:
+  - (1[012]|[1-9]):[0-5][0-9] (A\.M\.|P\.M\.)
+  - (1[012]|[1-9]):[0-5][0-9](?i)(a[\.]?m[\.]?|p[\.]?m[\.]?)
```

```suggestion
  - (1[012]|[1-9]):[0-5][0-9] (?i)(a\.m[^\.]|p\.m[^\.])
```
I couldn't figure out what this one was catching, but I think it was meant to catch no closing period?